integration.js

```diff
@@ -300,6 +301,34 @@ async function onMessage(payload, options, cb) {
         });
       }
       break;
+    case 'ADD_INTEGRATION_DATA_AS_NOTE':
+      Logger.info('Adding integration data as note', payload);
+      Logger.info('Options:', options.enableAddingNotes);
```

Comment:
These should not be logged at the `info` level and need to be removed or set to `trace` level logging.